server/logical.sql

```diff
@@ -0,0 +1,69 @@
+-- IF EXISTS(SELECT * FROM   dbo.users) DROP TABLE dbo.users;
+-- IF EXISTS(SELECT * FROM   dbo.cohort) DROP TABLE dbo.cohort;
+-- IF EXISTS(SELECT * FROM   dbo.events) DROP TABLE dbo.events;
+-- IF EXISTS(SELECT * FROM   dbo.recipes) DROP TABLE dbo.recipes;
+-- IF EXISTS(SELECT * FROM   dbo.dietary_restrictions) DROP TABLE dbo.dietary_restrictions;
+-- IF EXISTS(SELECT * FROM   dbo.allergies) DROP TABLE dbo.allergies;
+-- IF EXISTS(SELECT * FROM   dbo.recipe_allergies) DROP TABLE dbo.recipe_allergies;
```

Comment:
Hi John,
try this here:
DROP TABLE [IF EXISTS] table_name [CASCADE | RESTRICT];

Reply:
I should note:
here you should use CASCADE, not RESTRICT.
This will delete all dependencies and avoid any errors from dependencies.

see postgres docs for more info:
https://www.postgresql.org/docs/8.2/sql-droptable.html

All the best :D

Reply:
Hi Tom, I added the statements and pushed them.